crates/task-impls/src/quorum_proposal.rs

```diff
@@ -183,6 +182,7 @@ impl<TYPES: NodeType> ProposalDependencyHandle<TYPES> {
             &parent_leaf,
             commit_and_metadata.commitment,
             commit_and_metadata.metadata.clone(),
+            commit_and_metadata.fee_data.clone(),
```

Comment:
If we're moving `commit_and_metadata` do we need to clone this?

Reply:
We don't, thanks! Strange that clippy didn't notice.